images/win/scripts/Tests/Android.Tests.ps1

```diff
@@ -91,8 +95,12 @@ Describe "Android SDK" {
             "$installedPackages" | Should -Match $additionalToolVersion
         }
 
-        It "LTS NDK is installed" -TestCases @(@{ ndkLTSVersion = $ndkLTSMajorVersion; installedPackages = $androidInstalledPackages }) {
-            "$installedPackages" | Should -Match "ndk;$ndkLTSVersion"
+        It "NDK <ndkPackage> is installed" -TestCases $ndkPackagesTestCases {
+            "$installedPackages" | Should -Match "ndk;$ndkPackage"
+        }
+
+        It "Default NDK is installed" -TestCases @(@{ ndkDefaultVersion = $ndkDefaultMajorVersion; installedPackages = $androidInstalledPackages }) {
+            "$installedPackages" | Should -Match "ndk;$ndkDefaultVersion"
```

Comment:
Does it make sense to add a test to check that the ndk-bundle points to the default ndk version from toolset as we did for macOS and Ubuntu images?

Reply:
Added this test